extensions/gphl-proc-extension.dic

```diff
@@ -785,7 +785,9 @@ save__reflns.gphl_percent_possible_ellipsoidal_obs_anomalous
 
 save__reflns.gphl_redundancy_anomalous
     _item_description.description
-;              GPhL TBD
+;              The overall redundancy of anomalous data, i.e. where
```

Comment:
"data for which" instead of "where"?

Reply:
Note that all changes are here twice because the repository contains not just the base/extension files, but also the concatenated dist files, i.e. duplication. So everything below is identical to everything above ...